packages/ui/src/theme/overwrites/components/numberInput.ts

```diff
@@ -0,0 +1,38 @@
+export const NumberInput = {
+  sizes: {
+    sm: {
+      field: {
+        height: '32px',
+        padding: '1px 0px 1px 12px',
+      },
+    },
+    md: {
+      field: {
+        height: '40px',
+        padding: '1px 0px 1px 16px',
+      },
+    },
+    lg: {
+      field: {
+        height: '48px',
+        padding: '1px 0px 1px 16px',
+      },
+    },
+  },
+  variants: {
+    outline: {
+      field: {
+        borderRadius: '6px',
+        _focus: {
+          boxShadow: '0px 0px 0px 3px rgba(56, 189, 248, 0.6)',
+        },
+        _hover: {
+          border: '1px solid #9CA3AF',
```

Comment:
🤔 Is there a need to define border type and width? Maybe just specifying the `borderColor` property would be enough? Also, it would be great if you use the name of the color from our theme instead of a hex here